api-report/driver-utils.api.md

```diff
@@ -226,7 +226,7 @@ export class InsecureUrlResolver implements IUrlResolver {
 // @public
 export interface IProgress {
     cancel?: AbortSignal;
-    retry?(delayInMs: number, error: any): void;
+    onRetry?(delayInMs: number, error: any): void;
```

Comment:
Please double-check that it has no back-compa considerations. I.e. this interface is not crossing loader to driver boundary, or runtime to loader/driver boundary.

Reply:
yes, this is the only place IProgress is used today

Reply:
yes all usages are in loader only